jovian/utils/submit.py

```diff
@@ -0,0 +1,48 @@
+from jovian.utils.api import _h
+from jovian.utils.commit import commit, _parse_filename
+from jovian.utils.credentials import read_webapp_url
+from jovian.utils.error import ApiError
+from jovian.utils.logger import log
+from jovian.utils.misc import urljoin
+from jovian.utils.request import post, pretty
+from jovian.utils.shared import _u
+
+POST_API = '/learn/course/section/{}/make_submission'
+ASSIGNMENT_PAGE_URL = '/learn/{}/assignment/{}'
+
+
+def submit(assignment=None, notebook_url=None, **kwargs):
+    """ Performs jovian.commit and makes a assignment submission with the uploaded notebook.
+    """
+    if not assignment:
+        log("Please provide assignment", error=True)
+        return
+
+    filename = _parse_filename(kwargs.get('filename'))
+    if filename == '__notebook_source__.ipynb':
+        log("jovian.submit does not support kaggle notebook directly, make a commit serpartely and then do jovian.sunmit(notebook_url=<notebook_url_returned_after_commit>)", error=True)
```

Comment:
- typo in `jovian.submit`
- please add assignment arg as well
- use `"""` to split into multiple lines

Reply:
![image](https://user-images.githubusercontent.com/24864161/93863238-e7336380-fce0-11ea-86f8-fe0be0754327.png)
Preview